enhancements/update/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,197 @@
+Related to this is that it's currently unclear if the Cincinnati image itself will be published under its own product name or now.
+Further note: the repository for the graph-data images will not be semantically versioned.
+
+### Repository location and name
+Under which well-known container image repository will we publish the images?
+
+### Continuous build and release
+Can CPaaS handle continuous build and release from master?
+
+## Summary
+The [cincinnati-graph-data GitHub repository][cincinnati-graph-data] will be continuously packaged and released in form of a container image by the means of integration with CPaaS.
+This image will be available for public consumption, and will be used by the publicly available Cincinnati instances operated by Red Hat.
+The image can be transferred and served to Cincinnati instances operated by any party, including but not limited to network-restricted environments.
+The source of the container image, namely the cincinnati-graph-data, will remain publicly available and its workflows unaffected by this enhancement.
+
+## Motivation
+
+### Goals
+1. The container image will become the primary way of consuming and distributing the content of the cincinnati-graph-data repository.
+2. As long as the primary release metadata will be packaged, distributed and consumed in form of container images as well, the transport mechanism required for data consumption by Cincinnati, as well as mirroring and serving its consumed content, will be solely [Docker Registry HTTP API V2][docker-registry-http-api-v2] protocol.
+
+
+### Non-Goals
+1. Changes to the GitHub repository workflow or data format.
+2. Implementation details about the streamlining of the mirroring process for primary and secondary release metadata.
+
+## Proposal
+
+### User Stories
+
+#### Content mirroring for Cincinnati deployments
+A user might want to mirror all content which is required by Cincinnati.
+
+1. The user sets up a container image registry in the target environment.
+2. Optionally, if the target environment does not have internet access, the user downloads the release container images, the desired graph-data container image, and the images' verification signatures to a transfer medium.
+3. The user pushes the release, graph-data, and signature images to the target container image registry
+4. The user can now configure the Cincinnati instance in the target environment to consume the content from the available container image registry
+
+### Implementation Details/Notes/Constraints
+
+#### CPaaS Build configuration
+1. Every change to the master branch of the cincinnati-graph-data results in a new container image containing the unmodified working tree files of the new revision.
+  This will allow the plugin to continuously grab the latest version without being informed that a latest version exists.
+
+2. The image will be signed by an official Red Hat key, and a container image including the signature for the latest image will be produced.
```

Comment:
Link to [the "atomic container signature" spec][1]?

[1]: https://github.com/containers/image/blob/8a304be8982e1bb4151f46c7f3c68ea8233893eb/docs/containers-signature.5.md

Reply:
It's not clear to me yet how this signature will look like. Maybe it's worth looking at how the OCP release images are currently signed.